control-api/events.go

```diff
@@ -50,12 +50,12 @@ type NodeStoppedEvent struct {
 	Graceful bool   `json:"graceful"`
 }
 
-// TODO: remove omitempty in next version bump
 type HeartbeatEvent struct {
-	Version         string            `json:"version"`
-	NodeID          string            `json:"node_id"`
-	Nexus           string            `json:"nexus,omitempty"`
-	Uptime          string            `json:"uptime"`
-	Tags            map[string]string `json:"tags,omitempty"`
-	RunningMachines int               `json:"running_machines"`
+	AllowDuplicateWorkloads *bool             `json:"allow_duplicate_workloads"`
```

Comment:
Not a fan of using a pointer to a boolean here. Bool's zero value is false, which works fine